requirements/local.in

```diff
@@ -0,0 +1,2 @@
+# This is added so make requirements will auto install this edx-cookiecutter repository
+-e .
```

Comment:
@ reviewer: Adding this to keep backwards compatibility with how cookiecutters are made right now. Specifically, this allows python to find LayeredCookiecutter class in utils_edx_cookiecutters/layered_cookiecutter.py.  My current intention is for this to go away in a future PR and this repository becoming a cli package which can be pip installed and used to create our cookiecutters

Reply:
What would be the benefit of it becoming pip-installable?

Reply:
Ah, so this was the reason for adding `post-pip-compile.sh`.  Could we avoid this if we added `pip install -e .` to the end of `make requirements`?

Reply:
Yep, that is a much simpler option